src/main/java/avd/inf/jdm/rentmycar/domain/Ride.java

```diff
@@ -45,16 +45,10 @@ public void calculateBonusPointsForThisRide(Booking booking, User customer) {
         customer.setBonusPoints(customer.getBonusPoints() + (int)recklessnessToPoints);
     };
 
-    public Ride(User customer, Booking booking, double startRideLatitude, double startRideLongitude, double endRideLatitude, double endRideLongitude, double totalKilometersDriven, double maxAccelerationForce, LocalDateTime startDateTime, LocalDateTime endDateTime) {
+    public Ride (Booking booking) {
```

Comment:
We could set `totalKilometersDriven` to 0 on line 35 and `maxAccelerationForce` on line 36, if you want to give them some default values?

Reply:
![image](https://user-images.githubusercontent.com/36762061/196000166-e79bc48c-d361-45b6-bc4e-2e1a7b69da31.png)
its already saving to 0 after ending the ride,so fine with me like this :)